changelog/v0.36.0/bump-k8s-128.yaml

```diff
@@ -0,0 +1,51 @@
+changelog:
+- type: BREAKING_CHANGE
+  description: >
+    Bump dependencies to support Kubernetes 1.28
+  issueLink: https://github.com/solo-io/gloo/issues/8515
+  resolvesIssue: false
+- type: DEPENDENCY_BUMP
+  dependencyOwner: k8s.io
+  dependencyRepo: api
+  dependencyTag: v0.28.3
+- type: DEPENDENCY_BUMP
+  dependencyOwner: k8s.io
+  dependencyRepo: apiextensions-apiserver
+  dependencyTag: v0.28.3
+- type: DEPENDENCY_BUMP
+  dependencyOwner: k8s.io
+  dependencyRepo: apimachinery
+  dependencyTag: v0.28.3
+- type: DEPENDENCY_BUMP
+  dependencyOwner: k8s.io
+  dependencyRepo: client-go
+  dependencyTag: v0.28.3
+- type: DEPENDENCY_BUMP
+  dependencyOwner: sigs.k8s.io
+  dependencyRepo: code-generator
+  dependencyTag: v0.28.3
+- type: DEPENDENCY_BUMP
+  dependencyOwner: sigs.k8s.io
+  dependencyRepo: controller-runtime
+  dependencyTag: v0.16.3
+- type: DEPENDENCY_BUMP
+  dependencyOwner: onsi
+  dependencyRepo: ginkgo/v2
+  dependencyTag: v2.11.0
+- type: DEPENDENCY_BUMP
+  dependencyOwner: onsi
+  dependencyRepo: gomega
+  dependencyTag: v1.27.10
+- type: DEPENDENCY_BUMP
+  dependencyOwner: go.uber.org
+  dependencyRepo: zap
+  dependencyTag: v1.25.0
+
+- type: DEPENDENCY_BUMP
```

Comment:
the go.mod doesn't contain these changes

although when i tried to add them in, ran into codegen issues.
looks like they haven't been updated in years though. maybe we don't need the bumps?

Reply:
My bad, forgot to delete them after copying